src/DiffViewerProvider.ts

```diff
@@ -1,9 +1,9 @@
 import { Diff2HtmlConfig, parse } from "diff2html";
 import { LineMatchingType, OutputFormatType } from "diff2html/lib/types";
 import * as vscode from "vscode";
-import { DiffDocument } from "./DiffDocument";
+import * as path from "path";
 
-export class DiffViewerProvider implements vscode.CustomReadonlyEditorProvider<DiffDocument> {
+export class DiffViewerProvider implements vscode.CustomTextEditorProvider {
```

Comment:
TYVM! I've been meaning to change it to `CustomTextEditorProvider` for a while. :)